app/views/mailer/issue_edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if @journal.private_notes? %>
+  (<%= l(:field_private_notes) %>)
+<% end %>
+<%= l(:text_issue_updated, :id => link_to("##{@issue.id}", @issue_url), :author => h(@journal.user)).html_safe %>
+<hr />
+
+<ul class="journal details">
+<% details_to_strings(@journal_details, false, :only_path => false).each do |string| %>
+  <li><%= string %></li>
+<% end %>
+</ul>
+
+<% if Setting.plugin_redmine_advanced_messenger[:notifications_mail_option] == 'teaser' %>
+  <%= truncate(@journal.notes, length: 100, omission: '...') %>
```

Comment:
Here, I think we should also run `escape_javascript`. See comment in task.

Reply:
Done